resteasy-core/src/main/java/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -221,4 +240,150 @@ public ClassLoader run() {
       }
       return set.toArray(new Class<?>[]{});
    }
+
+   public class ContextOutputStream extends ServletOutputStream
```

Comment:
If we're going to make this an inner type it should be private.

Reply:
Done.